src/client/dfuse/ops/lookup.c

```diff
@@ -200,7 +192,7 @@ check_for_uns_ep(struct dfuse_projection_info *fs_handle,
 			else
 				DFUSE_TRA_WARNING(ie,
 						  "daos_pool_connect() failed, "
-						  DF_RC"\n", DP_RC(rc));
+						  DF_RC "\n", DP_RC(rc));
```

Comment:
Minor format question.
I thought our preferred style was to NOT use a space here?
```suggestion
						  DF_RC"\n", DP_RC(rc));
```

Reply:
You're entirely correct, this PR has been sitting a while.